### DISCUSSION TITLE
Can I know how to insallation cFS on Rapsberry pi??

### DISCUSSION THREAD

**Discussion by WOODUCK** [nasa/cFS#478] [Q&A]
hello

I am installing the cFS on raspberry pi. But I have trouble with installation.

The sequence of installation is 
git clone https://github.com/nasa/cFS.git
→ cd cFS
→ git submodule init
→ git submodule update
→ cp cfe/cmake/Makefile.sample Makefile
→ cp -r cfe/cmake/sample_defs sample_defs

when I typed "make prep", there was a problem.

Can I know the reason why??

I am looking forward to your reply.

Thanks for looking at my question.

![제목 없음](https://user-images.githubusercontent.com/86736519/167824188-87b2228e-2c53-4f45-8082-6f8ec9c1b3ab.jpg)

**Comment by HugoValente11** [comment]
As seen in the error message you don't have cmake
You should install cmake and any other dependencies needed.
For cmake you can run: `sudo apt install cmake`
